src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -94,6 +98,21 @@ drop-*≡* (*≡* eq) = eq
 _≃?_ : Decidable _≃_
 p ≃? q = Dec.map′ *≡* drop-*≡* (↥ p ℤ.* ↧ q ℤ.≟ ↥ q ℤ.* ↧ p)
 
+0ℚᵘ≠1ℚᵘ : 0ℚᵘ ≠ 1ℚᵘ
```

Comment:
I feel like this could probably be just `0≠1`?

Reply:
I renamed to 0≠1